### DISCUSSION TITLE
College email related issue.

### DISCUSSION THREAD

**Discussion by theharshjha** [community/community#109566] [GitHub Education]
### Hi everyone,

Hi everyone,
I don't have a student email provided by the college. How can I apply for the GitHub student developer pack?

**Comment by nancyibrahim22** [comment]
Try to put your personal mail, but you have to put your school's name and any proof that you are a student in this school (screenshot of your ID or proof of enrollment letter) and I hope it works

**Reply by theharshjha** [reply]
![image](https://github.com/community/community/assets/83906334/f2938834-19af-408c-9d8f-beb2aabb916f)
didn't find any input field or link for that.

**Reply by nancyibrahim22** [reply]
try this tutorial: https://youtu.be/xCpNu7ElZdU?si=7BHaIYW03rMkUXwo
